client/src/offline-settings/db.ts

```diff
@@ -0,0 +1,145 @@
+// WARNING - This file is duplicated at two locations:
```

Comment:
We should add a ticket to remove the migration logic after 2 weeks (then anyone that didn't migrate has to re-download all content anyhow).

Reply:
Added to our board.